### ISSUE TITLE
Offscreen's `paint` rendering event has wrong parameter `dirty` area

### ISSUE THREAD

**Issue by MikeAski** [electron/electron#12447]
* Electron version: 1.7 (locked on this version as latest one breaks one dependency)
* Operating system: Mac OS X / Ubuntu Linux Xenix

### Expected behavior

Registered callback on `paint` event should receive the updated "dirty" area.

### Actual behavior

Dirty object has either random geometry, either covers the whole page's content area.

### How to reproduce

Clone https://github.com/MikeAski/electron-paint-issue and run:
```
yarn
yarn reproduce
```

**Comment by brenca** [comment]
I'm not sure how we could resolve this. Those dirty rects come from the guts of chromium itself.

Do you see the same behavior with `app.disableHardwareAcceleration()` too?

**Comment by MikeAski** [comment]
@brenca Yep, absolutely. Same behaviour without or without HW acceleration...

**Comment by sofianguy** [comment]
Thank you for taking the time to report this issue and helping to make Electron better.

The version of Electron you reported this on has been superseded by newer releases. 

If you're still experiencing this issue in Electron v2.0.x or later, please add a comment specifying the version you're testing with and any other new information that a maintainer trying to reproduce the issue should know.

I'm setting the `blocked/need-info` label for the above reasons.

Thanks in advance! Your help is appreciated.
